src/GoogleFit/FitOnFhir.GoogleFit.Tests/GoogleFitAuthorizationHandlerTests.cs

```diff
@@ -4,6 +4,7 @@
 // -------------------------------------------------------------------------------------------------
 
 using System.Net;
+using Azure.Core;
```

Comment:
It looks like Azure.Core and Microsoft.Extensions.Logging are not needed and they are causing a compile error on my machine.